src/PolyhedralGeometry/markov_basis.jl

```diff
@@ -0,0 +1,68 @@
+@doc Markdown.doc"""
+    markov_basis(S; use_kernel=true)
+    markov_basis(P::Polyhedron{fmpq})
+
+
+Return a Markov basis for the lattice $L$. Where $L$ is
+- either the integer kernel of a matrix when either `use_kernel=true` or the
+  argument is of type `Polyhedron{fmpq}`,
+- otherwise $L$ is given by the span of the collection of integer vectors `S`.
+# Arguments
+- `S::AbstractVector{<:AbstractVector{U}} where U <: Union{Base.Integer, fmpz}`:
```

Comment:
Please use `IntegerUnion`.

This type description should be at the start of the docstring, i.e., replace
```
    markov_basis(S; use_kernel=true)
    markov_basis(P::Polyhedron{fmpq})
```
by something like this (I am guessing a bit here)
```
    markov_basis(S::AbstractVector{<:AbstractVector{<:IntegerUnion}})
    markov_basis(S::MatElem{<:IntegerUnion}; use_kernel::Bool=true)
    markov_basis(S::AbstractMatrix{<:IntegerUnion}; use_kernel::Bool=true)
    markov_basis(P::Polyhedron{fmpq})
```